radix-engine/src/kernel/kernel_client_facade.rs

```diff
@@ -438,6 +438,10 @@ where
     fn update_instruction_index(&mut self, new_index: usize) -> Result<(), RuntimeError> {
         KernelModuleMixer::on_update_instruction_index(self, new_index)
     }
+
+    fn wasm_memory_consumption(&mut self, size: usize) {
+        self.current_frame.consumed_wasm_memory = size;
```

Comment:
ClientAPI implementation shouldn't be accessing `current_frame` directly. You will find this stop working shortly after merging https://github.com/radixdlt/radixdlt-scrypto/pull/815

Reply:
Changed that to use `KernelModuleMixer::on_update_wasm_memory_usage`.